packages/server/server/api/destinations/destination.service.ts

```diff
@@ -15,7 +15,7 @@ class DestinationService {
             .then((res) => {
                 if (res) {
                     logger.info(`destination with placeId: ${res.placeId} already exists`);
-                    return this.updateDestinationById(res.uuid, destData);
+                    return Promise.resolve(res);
```

Comment:
This is what was causing the bug with `Save` to trecipe from Destination Page I mentioned in slack. `updateDestinationById` used to update destination uuids as well, which we don't want (because many other trecipes refer to this uuid and depends on it being same).